src/coreComponents/common/CMakeLists.txt

```diff
@@ -33,24 +33,12 @@ set( common_sources
      initializeEnvironment.cpp
    )
 
-set( dependencyList lvarray pugixml RAJA chai conduit::conduit fmt::fmt )
+set( dependencyList ${commonDeps} lvarray pugixml::pugixml RAJA chai conduit::conduit fmt::fmt )
```

Comment:
what is up with the scoping here?

Reply:
Between pugixml v1.8 (currently in our tpls) and v1.11.4 (installed by spack on crusher), they added scoping to heir exported targets, similar to conduit and fmt later in the line.

I think this is best handled in the tpl discovery scripts, so we can use our tpls uniformly throughout the rest of the codebase without scoping. I'll add logic there to our currently scoped tpl import targets to alias them as unscoped for consistency.